ansible_ai_connect_chatbot/src/useChatbot/useChatbot.ts

```diff
@@ -212,9 +212,25 @@ export const useChatbot = () => {
         onClick: () => {
           if (message.actions) {
             message.actions.copy.className = "action-button-clicked";
-            navigator.clipboard.writeText(
-              typeof response === "object" ? response.response : response,
-            );
+            const ref_docs =
+              typeof response === "object"
+                ? response.referenced_documents?.slice(0, 30)
```

Comment:
What this `.slice(0, 30)` is for? I was assuming that the maximum size of  `response.referenced_documents` is `5`...

Reply:
@TamiTakamiya I was not aware of the maximum size of `response.referenced_documents` that's why put a higher number. Can you please point me to the information around the maximum size of `response.referenced_documents`. Thanks